sqlmesh/dbt/model.py

```diff
@@ -98,6 +101,14 @@ def _validate_list(cls, v: t.Union[str, t.List[str]]) -> t.List[str]:
     def _validate_sql(cls, v: t.Union[str, SqlStr]) -> SqlStr:
         return SqlStr(v)
 
+    @validator("partition_by", pre=True)
+    def _validate_partition_by(cls, v: t.Dict[str, t.Any]) -> t.Dict[str, t.Any]:
+        if not v.get("field"):
+            raise ConfigError("'field' key required for partition_by.")
```

Comment:
This only true for BigQuery. For example in Spark you can just specify a list of strings
https://docs.getdbt.com/reference/resource-configs/spark-configs
```
{{ config(
    materialized='incremental',
    partition_by=['date_day'],
    file_format='parquet'
) }}
```

Reply:
Thanks for pointing out. I didn't realize dbt spark had the same field with a different format.